tests/DoctrineMongoODMModuleTest/Doctrine/ConnectionFactoryTest.php

```diff
@@ -174,4 +175,25 @@ public function testConnectionStringWithOptionsShouldSetDefaultDB()
 
         self::assertEquals($dbName, $configuration->getDefaultDB());
     }
+
+    public function testEventManagerShouldSetEventManager()
```

Comment:
What do you mean, event manager should set event manager? Don't you mean connection factory should set event manager?